google/cloud/firestore_v1/base_collection.py

```diff
@@ -124,7 +124,10 @@ def document(self, document_id: str = None) -> DocumentReference:
         if document_id is None:
             document_id = _auto_id()
 
-        child_path = self._path + (document_id,)
+        # Append `self._path` and the passed document's  Id as long as the first
```

Comment:
```suggestion
        # Append `self._path` and the passed document's ID as long as the first
```

nit: extra space, Id -> ID